components/tyk-dev-portal/templates/statefulset-enterprise-portal.yaml

```diff
@@ -168,7 +168,7 @@ spec:
         resources:
 {{ toYaml .Values.resources | indent 12 }}
         command: ["/opt/portal/dev-portal"]
-        {{- if .Values.global.bootstrap.devPortal }}
+        {{- if .Values.global.components.bootstrap }}
```

Comment:
I think it needs to check both if the global bootstrapping is enabled and dev portal bootstrapping is enabled.